cli/src/pcluster/config/cluster_config.py

```diff
@@ -97,6 +97,7 @@
     RootVolumeEncryptionConsistencyValidator,
     RootVolumeSizeValidator,
     SchedulableMemoryValidator,
+    SchedulerDisableSudoAccessForDefaultUserValidator,
```

Comment:
Why the validator is related to the scheduler? Isn't this disablement cluster-wide?

Reply:
I am confused as why the validator should be cluster-wide?

Reply:
I think the validator should not be there because the configuration is cluster-wide so there is no reason to consider it as not appliable to a aws batch cluster (in a aws batch cluster you would apply the sudoer removal in the head node)